third_party/cppmicroservices_elf.h

```diff
@@ -37,32 +37,32 @@
 #include <cstdint>
 
 /* Type for a 16-bit quantity.  */
-typedef uint16_t Elf32_Half;
-typedef uint16_t Elf64_Half;
+using Elf32_Half = uint16_t;
```

Comment:
I think we should not "modernize" third-party files. It makes it harder to keep track of our real changes to the original files.

Reply:
I 100% agree. I'll undo the changes made to all the third-party files.